pkg/generate/app/cmd/newapp_test.go

```diff
@@ -1265,6 +1265,11 @@ func TestEnsureValidUniqueName(t *testing.T) {
 			expected: []string{"one", "one-1", "two"},
 		},
 		{
+			name:     "non-standart characters",
+			input:    []string{"Emby.One", "test-_test", "_-_"},
+			expected: []string{"embyone", "test-test", ""},
```

Comment:
Is it possible to have repo on GitHub with `@-MyRepo` name? Or it's not only for GitHub repositories?